framework7/dom7.d.ts

```diff
@@ -0,0 +1,212 @@
+// Type definitions for Dom7
+// Project: https://github.com/nolimits4web/Framework7
+// Definitions by: JasonKleban <https://github.com/JasonKleban/Framework7.d.ts>
```

Comment:
I think you mean: `// Definitions by: JasonKleban <https://github.com/JasonKleban>`